build.gradle

```diff
@@ -57,7 +57,7 @@ subprojects {
     intellij {
         type = ideaEdition
         version = ideaVersion
-        updateSinceUntilBuild = true
+        updateSinceUntilBuild = false
```

Comment:
This will prevent gradle-intellij-plugin from overriding our hardcoded versions in plugin.xml